TCMSApp/src/client/test-helpers/mock-data.js

```diff
@@ -31,45 +103,568 @@ var mockData = (function () {
         // Code here will be linted with JSHint.
         /* jshint ignore:start */
         return [
```

Comment:
actually it was better in folded state :) we even had special lines to allow the long lines:

`// Code here will be linted with JSHint.
          /* jshint ignore:start */`

why have you leaved this comments?